utils/sendMail.ts

```diff
@@ -0,0 +1,21 @@
+import { sendEmail } from "@nodemailer/mail";
```

Comment:
We have `services/nodemailer.mail.ts`. Should we get rid of that if we're going to use this?

Reply:
Done